core/domain/summary_services.py

```diff
@@ -23,6 +23,17 @@
 import utils
 
 
+def get_human_readable_contributors_summary(contributors_summary):
+    contributors_items = contributors_summary.items()
```

Comment:
I think it's a bit clearer here to use

```
    contributor_ids = contributors_summary.keys()
```
since then you avoid the use of [0], [1]. And then, below, you can do

```
return {
    contributor_id: contributor_usernames[ind]
    for ind in range(len(contributor_ids))
}
```